public/script.js

```diff
@@ -458,7 +458,9 @@ let currentVersion = '0.0.0';
 export const default_ch_mes = 'Hello';
 let generatedPromptCache = '';
 let generation_started = new Date();
+/** @type {v1CharData[]} */
 export let characters = [];
+/** @type {number | undefined} */
```

Comment:
Not entirely true as under some circumstances it could also be a string... Better leave it untyped here.

Reply:
https://github.com/SillyTavern/SillyTavern/commit/3988038623854b1871035391563e2664b32d27db

Reply:
This still causes type errors so I will remove it for now. You can look into fixing it separately.